azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -29,6 +29,11 @@ resources:
     name: AzureAD/azure-activedirectory-library-for-android
     ref: dev
     endpoint: ANDROID_GITHUB
+  - repository: OneAuth
+    type: git
+    name: OneAuth/OneAuthAndroidCommonWrapper
+    ref: main
+    endpoint: OneAuthServiceConnection_2
```

Comment:
can we use the same ANDROID_GITHUB service connection for this, so that we only need to update one connection when its close to expiry/expired

Reply:
this one auth connection is to connection to the url of visual studio, I do not see that in ANDROID_GITHUB, as Oneauth does not reside on Github